src/foam/nanos/logger/StringBufferLogger.java

```diff
@@ -0,0 +1,80 @@
+package foam.nanos.logger;
+
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Formatter;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+public class StringBufferLogger extends AbstractLogger {
+
+  protected java.util.logging.Logger logger_; 
+  
+  public StringBufferLogger() {
+    logger_ = java.util.logging.Logger.getAnonymousLogger();
+    logger_.setUseParentHandlers(false);
+    logger_.setLevel(Level.ALL);
+    Handler handler = new ConsoleHandler();

```

Comment:
Does this mean that the output is also going to the console? Do we need to use java.util.Logger? Does it provide any value in this case?

Reply:
Does this mean that the output is also going to the console? yes
Do we need to use java.util.Logger? yes, I follow the same principle as StdoutLogger
Does it provide any value in this case?

Reply:
I think the use case for this is capturing logging from Scripts. Is that correct? If so, I don't think we also want the output to go to the stdout.